net-dns/knot/knot-3.2.2.ebuild

```diff
@@ -69,6 +73,7 @@ src_configure() {
 		$(use_enable fastparser)
 		$(use_enable dnstap)
 		$(use_enable doc documentation)
+		$(use_enable quic)
 		$(use_enable utils utilities)
 		$(use_enable xdp)
 		--enable-systemd=$(usex systemd)
```

Comment:
I added `$(use_enable caps cap_ng)` here because we still had an automagic dependency on libcap-ng.